js_modules/dagit/packages/ui/src/components/ProductTour.tsx

```diff
@@ -31,7 +31,7 @@ type Props = {
     next?: () => void;
     dismiss?: () => void;
   };
-  position: Position;
+  position: ProductTourPosition;
```

Comment:
Could `ProductTour` just use the same placement type/values as `Popover`?

Reply:
Yeah I was thinking about doing that but was a little torn about whether or not to use Popover because the arrow is always pointing at the center and we wanted to be able to have it point to the left/right as well as the center :\

Reply:
Lets make a follow up task to fix this